src/main/java/org/sefglobal/scholarx/config/SecurityConfig.java

```diff
@@ -78,6 +88,29 @@ public void onLogoutSuccess(HttpServletRequest request,
       .accessTokenResponseClient(authorizationCodeTokenResponseClient());
   }
 
+  @Bean
+  public CorsConfigurationSource corsConfigurationSource() {
+    final CorsConfiguration configuration = new CorsConfiguration();
+
+    configuration.setAllowedOrigins(ImmutableList.of("*"));
```

Comment:
Doesn't this line allow any origin to access the API?
I even took a pull from your `develop` branch and tested it out, it allows cross-origin resource sharing for any port now.
Shall we remove this line? 
And don't forget to test the implementation out for some other origins apart from `:3000` as well :)

Reply:
@Gravewalker666 thank you for pointing that out. I've added that line by mistake. Removed the line in the commit as suggested. Tested in different ports and on Chrome and Firefox. Works as expected now.
Will keep that convention in mind ;)
Thank you and please review again when you can :)

Reply:
Nice it works fine now. 
I see you have accidentally messed up the git commit timeline a bit, the rule of thumb is "One commit per one PR"
That is because we are trying to follow a practice wherewith one PR only one change is done. This helps us to keep PR simple and short. 
I'm sorry for annoying you by requesting changes on additional stuff this much, but can you please remove the unnecessary commits? :)
And I'm sorry if I wasn't being clear with the last comment, the commit message should be able to fit that particular sentence. That sentence should not be included in the commit message.

Reply:
Sorry about the mess-up. I cleaned up the commit timeline and adjusted the commit message. Hope everything is good now. Thanks for all the help :)

Reply:
Nice, thanks for getting this done dude. Nice work 👍